src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -572,6 +570,21 @@ def visit_span(self, span: Span):
                 offender_span_ids=spans_involved,
             )
 
+    @classmethod
+    def is_span_eligible(cls, span: Span) -> bool:
+        description = span.get("description", None)
+        if not description:
+            return False
+
+        description = description.strip()
+        if description[:6].upper() != "SELECT":
```

Comment:
I think startswith would be the same?

Reply:
Sometimes the query starts with a lowercase `select` so I can't really use `startswith` here (I guess I could but then I'd need to check with `upper()` and `lower()`)

Reply:
Fair enough! I personally like doing upper on the whole description, will leave it to you tho!